ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Tables/CVEsTable.tsx

```diff
@@ -100,14 +104,33 @@ function CVEsTable({
     getSortParams,
     isFiltered,
     filteredSeverities,
+    showExceptionMenuItems,
+    selectedCves,
+    cveTableActionHandler,
 }: CVEsTableProps) {
     const expandedRowSet = useSet<string>();
 
+    const colSpan = 6 + (showExceptionMenuItems ? 2 : 0);
+
     return (
         <TableComposable borders={false} variant="compact">
             <Thead noWrap>
                 <Tr>
                     <Th>{/* Header for expanded column */}</Th>
+                    {showExceptionMenuItems && (
+                        <Th
+                            title={
+                                selectedCves.size > 0
+                                    ? `Clear ${pluralize(selectedCves.size, 'selected CVE')}`
+                                    : undefined
+                            }
+                            select={{
+                                isSelected: selectedCves.size > 0,
+                                isDisabled: selectedCves.size === 0,
```

Comment:
What are pro and con of synonym to emphasize contrast between:
* `size !== 0`
* `size === 0`